File: TAPPlayground/SessionManagement/Models/Product.cs
namespace SessionManagement.Models;

public class Product{
    public int Id {get;set;}

    public String? Title{get;set;}
    public String? Description{get;set;}

    public int StockAvailable{get;set;}

    public double UnitPrice{get;set;}
    
    public string? Image{get; set;}

}

File: TAPPlayground/TFLNGApp/src/app/Catalog/product.ts

export class Product{

    private title:string;
    private description:string;
    private quantity:number;
    private unitPrice:number;
    private imageUrl:string;
    public  likes:number;

    //No contructor overloading

    constructor(){
        this.title="Gerbera";
        this.description="Wedding Flower";
        this.quantity=5000;
        this.unitPrice=10;
        this.imageUrl="./assets/images/gerbera.jpg";
        this.likes=1200;
    }
}